api/api-doc/storage_service.json

```diff
@@ -1122,6 +1122,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"small_table_optimization",
+                     "description":"If the value is the string 'true' with any capitalization, perform small table optimization.",
```

Comment:
```suggestion
                     "description":"If the value of the string 'true' with any capitalization, perform small table optimization.",
```

Reply:
I think changing to "of" is not a good idea.

Reply:
On a related note: we should also accept `True`, `yes`, `1` as `true`.

Reply:
It is not like db/config.cc. The bool_opt in repair only supports true/True/TRUE. We could extend it but it is not important for now.

Reply:
I think mykaul's suggestion is wrong. You can say "if the value **is** the string true"  (what you wrote) or "if the value **of** the string **is** true, but not Yaniv's suggestion with only of and no is.